gridsome.server.js

```diff
@@ -5,12 +5,26 @@
 // Changes here require a server restart.
 // To restart press CTRL + C in terminal and run `gridsome develop`
 
-module.exports = function (api) {
-  api.loadSource(({ addContentType }) => {
-    // Use the Data Store API here: https://gridsome.org/docs/data-store-api
-  })
+const axios = require("axios");
 
-  api.createPages(({ createPage }) => {
-    // Use the Pages API here: https://gridsome.org/docs/pages-api
-  })
-}
+module.exports = function(api) {
+  api.loadSource(async store => {
```

Comment:
Is this going to break things on the server? I presume everything in `gridsome.config.js` is all we need for airtable, and that we can remove this part

Reply:
@experimatt It does not break things on the server. Do you have the Netlify login creds? But yes go ahead and remove it next time you get a chance. I will not get a chance to touch any of this til late in the weekend.

Reply:
It's OK I haven't had a chance to look at anything since we did our hacking weekend two weeks agin 🙈